### PR TITLE
Minor fixes for channel-about page

### PR DESCRIPTION
**Pull request by btzr-io**
## PR Checklist

Please check all that apply to this PR using "x":

- [X] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

- Render Markdown in channel descriptions #2506
- Missing external links on channel page #2519
- Channel description word cutoff #2485

### CODE REVIEW

**Comment by seanyesmunt** [comment]
Thanks for this! I've been meaning to get to it.

File: src/ui/scss/component/_media.scss [hunk 1]
```diff
@@ -260,7 +260,6 @@
 
 .media__info-text {
   font-size: 1.15rem;
-  word-break: break-all;
```
[the remark this answers is not in the document]
Comment (btzr-io) [reply src/ui/scss/component/_media.scss]: Oh, I see, then I think `word-break: break-word` should work :+1:

[the remark this answers is not in the document]
Comment (btzr-io) [reply src/ui/scss/component/_media.scss]: @seanyesmunt How long it should be the `max-width` ?

[the remark this answers is not in the document]
Comment (btzr-io) [reply src/ui/scss/component/_media.scss]: I added 80% as a random value, looks good in my screen :stuck_out_tongue:

File: src/ui/scss/component/_media.scss [hunk 2]
```diff
@@ -260,11 +260,16 @@
...
 .media__info-text {
   font-size: 1.15rem;
+  word-break: break-word;
 
   &:not(:last-of-type) {
     margin-bottom: var(--spacing-vertical-large);
   }
 
+  &.media__info-text--small {
+    max-width: 70%;
```
Comment (btzr-io) [inline src/ui/scss/component/_media.scss]: @seanyesmunt let me know if you have a specific value in mind :v:

Comment (seanyesmunt) [reply src/ui/scss/component/_media.scss]: 50rem seems to work pretty well.

Comment (btzr-io) [reply src/ui/scss/component/_media.scss]: Ok done!

**Pull request merged**